docs/build/getting-started/networks-endpoints.mdx

```diff
@@ -41,6 +41,18 @@ This network runs under a [Coordinator PoA scheme](/learn/protocols/coordinator/
 
 <NetworkInfo.L1 {...Networks['iota']}/>
 
+### IOTA EVM
```

Comment:
Why is this in this PR?

Reply:
Refactored all the branches. and updated them with the correct merges. So no it is not part of it anymore ;)